cmd/routed-eni-cni-plugin/driver/driver.go

```diff
@@ -208,6 +211,12 @@ func (createVethContext *createVethPairContext) run(hostNS ns.NetNS) error {
 		return errors.Wrap(err, "setup NS network: failed to add static ARP")
 	}
 
+	if createVethContext.v6Addr != nil && createVethContext.v6Addr.IP.To16() != nil {
+		if err := WaitForAddressesToBeStable(createVethContext.contVethName, 10); err != nil {
```

Comment:
1. It's weird that you have WAIT_INTERVAL as a constant above, and then hardcode '10' here, even though this '10' is much more important :stuck_out_tongue: 
2. Don't just use `10` here, instead this function should take an arg of `time.Duration`.  Getting confused about time units (seconds? milliseconds? hours?) is a really common bug, so make that impossible:

```
const DADTimeout = 10 * time.Second

// ... here
		if err := WaitForAddressesToBeStable(createVethContext.contVethName, DADTimeout); err != nil {
		
// and below:
func WaitForAddressesToBeStable(ifName string, timeout time.Duration) error {
        deadline := time.Now().Add(timeout)
        // ...
}
```

Reply:
Yeah, fair enough. Will make the change.